### ISSUE TITLE
StaticClosure sniff does not detect $this in string

### ISSUE THREAD

**Issue by arxeiss** [slevomat/coding-standard#807]
In older project I'm trying to add your rules into PHPCS. However I have a problem with **StaticClosure sniff**. In the following method sniff tries to add `static` keyword before closure. 

However it is not valid  after that change. Because sniff does not detect `$this` when it is in the string.

```php
$this->searchOperator = function ($query) {
        $query->raw("DATE_FORMAT(`{$this->columnName}`, '%Y-%m-%d')");
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/80301b06e58276c06cc4a0d0e391142a805ebb8d
